src/pages/sym.mdx

```diff
@@ -0,0 +1,133 @@
+The character encoding must be well-formed UTF-8 (without BOM), however it is strongly encouraged to stick to ASCII characters for portability.
+The NUL character (`\0`, U+0000) shall not appear anywhere in the file.
+An implementation encountering one must output a diagnostic if it can, and may then ignore it, the line, the whole file, or reject the file outright.
+
+Especially since they are intended to be somewhat human-readable, sym files can contain comments.
+Comments begin with a semicolon anywhere in a line, all the way to the end of that line.
+Semicolons cannot be escaped, as they are not valid in any part of entries.
+Comments should be ignored by the implementation.
+
+After removing the comment (if any), the line shall be split into non-empty **tokens**, separated by one or more consecutive **whitespace** characters.
+ASCII spaces (` `, U+0020) must be treated as whitespace; implementations may also treat tabs (`\t`, U+0009) as whitespace, but this is not portable to some older implementations.
+Other Unicode "White Space" characters must *not* be treated as whitespace.
+Leading or trailing whitespace must be ignored.
+
+### Line processing
+
+If any part of a line cannot be processed by the implementation, it (and possibly the whole line) shall be ignored, and a warning should be produced.
+
+A line without any tokens shall be silently ignored, to permit empty lines, even if they contain whitespace and/or a comment.
+
+A line with only one token shall be ignored, being reserved for future extensions.
+Encountering one may produce a warning.
+
+Otherwise, the line defines a single **symbol**: the first token provides its location, the second token provides its name, and further tokens (if any) may provide extra metadata.
+Extra tokens are currently not specified, but may be so in the future; so, implementations should ignore them, and may produce a warning in doing so.
+
+#### Symbols
+
+A symbol is uniquely defined by the combination of its location and its name; symbol names are case-sensitive.
+If the same symbol (identical name *and* identical location) is defined twice, one of the two definitions must be silently ignored.
+
+If two distinct symbols are defined with the same name (but different locations), neither definition shall be ignored, and this should not be an error.
+A warning may be produced, though not necessarily when loading the sym file: for example, "displaying names instead of memory addresses" accomodates this case just fine, however a "go to symbol" feature wouldn't, and a warning could be produced when the user tries to go to an ambiguous target.
+
+##### Location
+
+The location shall be of the form `<bank>:<address>`, both `<bank>` and `<address>` being case-insensitive hexadecimal numbers, possibly zero-padded, but without any prefix (so `$02:4000` is not a valid location, for example).
```

Comment:
```suggestion
The location shall be of the form `<bank>:<address>`, both `<bank>` and `<address>` being case-insensitive hexadecimal digits, possibly zero-padded, separated by a colon (`;` U+0058). Typical hexadecimal prefixes (`0x`, `$`, etc) are not allowed.
```

Reply:
It'd have to be "composed of" to use "digits".

Reply:
Edited.